docs/setup.asciidoc

```diff
@@ -145,3 +146,11 @@ For example, in an ASP.NET MVC application, you can place the above call in the
 NOTE: Be careful not to execute `DbInterception.Add` for the same interceptor more than once,
 or you'll get additional interceptor instances.
 For example, if you add `Ef6Interceptor` interceptor twice, you'll see two spans for every SQL query. 
+
+[float]
+[[setup-general]]
+==== Other .NET applications
+
+In case you have a .NET application which is not covered above, you can still use the agent and instrument your application manually. 
```

Comment:
For this to work, the only requirement is that they run on a .NET Flavor that supports .NET Standard 2.0. 

But this is covered on the `Supported technologies` page, so I feel we can skip it here.